### PR TITLE
utils_kernel_module.py: unload_module method improvement

### PR DESCRIPTION
**Pull request by PaulYuuu**
1. `unload_module` should also unload the module's holders
2. Some modules don't have parameters, so they will not have `parameters` directory

ID: 1401

### CODE REVIEW

**Comment by PaulYuuu** [comment]
With this PR:
```bash
# lsmod | grep vfio
vfio_pci              262144  0
vfio_pci_core         327680  1 vfio_pci
vfio_iommu_type1      327680  0
vfio                  327680  3 vfio_pci_core,vfio_iommu_type1,vfio_pci
```
```python
>>> from virttest import utils_kernel_module
>>> vfio = utils_kernel_module.KernelModuleHandler('vfio')
>>> vfio.unload_module()
```
```bash
# lsmod | grep vfio

```
```python
vfio.restore()
```
```bash
# lsmod | grep vfio
vfio_pci              262144  0
vfio_pci_core         327680  1 vfio_pci
vfio_iommu_type1      327680  0
vfio                  327680  3 vfio_pci_core,vfio_iommu_type1,vfio_pci
```


Without this PR:
```python
>>> from virttest import utils_kernel_module
>>> vfio = utils_kernel_module.KernelModuleHandler('vfio')
>>> vfio.unload_module()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/yihyu/kar/workspace/avocado-vt/virttest/utils_kernel_module.py", line 104, in unload_module
    raise KernelModuleUnloadError(self._module_name, output)
virttest.utils_kernel_module.KernelModuleUnloadError: Couldn't unload module vfio:
```

**Comment by PaulYuuu** [comment]
Hello @YongxueHong @smitterl, would you help to review this one? Thanks.

**Review by smitterl — APPROVED** [review]
Thanks for this, LGTM

File: virttest/utils_kernel_module.py [hunk 1]
```diff
@@ -232,22 +229,26 @@ def current_config(self):
...
         """
 
-        if not os.path.exists(self._module_params_path):
-            return None
+        # Module not loaded
+        if not os.path.exists(self._module_path):
+            return
+        # Some modules do not have parameters
+        elif not os.path.exists(self._module_params_path):
+            return ''
```
[the remark this answers is not in the document]
Comment (PaulYuuu) [reply virttest/utils_kernel_module.py:243]: Here, if `return None`, means the module is not loaded, and `return ''` means this module is loaded but no any parameters.

[the remark this answers is not in the document]
Comment (YongxueHong) [reply virttest/utils_kernel_module.py:243]: > Here, if `return None`, means the module is not loaded, and `return ''` means this module is loaded but no any parameters.

Okay, got your point. Thanks.

**Review by YongxueHong — APPROVED** [review]

**Pull request merged**